src/palace/manager/feed/annotator/base.py

```diff
@@ -243,6 +244,21 @@ def content(cls, work: Work | None) -> str:
 
 
 class Annotator(ToFeedEntry):
+    @classmethod
+    def _sample_link_comparator(cls, link1: Link, link2: Link):
```

Comment:
Since `cls` isn't used, this could be a `@staticmethod` or pulled out of the class as a function.

Reply:
Also, we need a return type annotation.